t/unit/utils/test_time.py

```diff
@@ -253,3 +254,41 @@ def test_utcoffset(self, patching):
         assert utcoffset(time=_time) is not None
         _time.daylight = False
         assert utcoffset(time=_time) is not None
+
+
+class test_get_exponential_backoff_interval:
+
+    @patch('random.randrange', lambda n: n - 2)
```

Comment:
Is there a reason why you're mocking `randrange` as `lambda n: n - 2` here, but mocking it as `lambda n: n - 1` in the `test_negative_values` test? It's not _wrong_, but it's confusing.